SSSpinnerButton/SpinnerButton/SSSpinnerButton.swift

```diff
@@ -466,3 +476,13 @@ private extension SSSpinnerButton {
     }
     
 }
+
+
+extension SSSpinnerButton {
+    
+    public struct Config {
```

Comment:
structure name can be more meaningful.

Reply:
It's inside the extension, so it is meaningful, `SSSpinnerButton. Config`